payloads/ready.go

```diff
@@ -45,6 +45,10 @@ type Ready struct {
 	// cpu[0-9]+ entries in /proc/stat.
 	CpusOnline int `yaml:"cpus_online"`
 
+	// Array containing one entry for each network interface present on the
+	// CN/NN
+	Networks []NetworkStat
```

Comment:
@tpepper The launcher today only reports compute networks. The networking library provides both to launcher.